### ISSUE TITLE
Make inlay hints accessible

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#142532]
Inlay hints are currently not accessible and should define model for that

**Comment by jrieken** [comment]
First version is done: there is start/stop read line with hints commands

<img width="1338" alt="Screen Shot 2022-02-11 at 11 15 47" src="https://user-images.githubusercontent.com/1794099/153576614-a1726879-fff1-4240-98fe-2b0e41789ef1.png">

**Comment by jrieken** [comment]
To verify

* enable screen reading
* select F1 > Read Line with Inlay Hint
* ensure the line is read with its inlay hints
* ensure ESC brings you back to the editor
* ensure nothing happens you there are no hints on the line
